sharding/nodesCoordinator/indexHashedNodesCoordinatorLite_test.go

```diff
@@ -0,0 +1,89 @@
+package nodesCoordinator
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestIndexHashedNodesCoordinator_SetNodesConfigFromValidatorsInfo(t *testing.T) {
+	t.Parallel()
+
+	arguments := createArguments()
+
+	ihgs, err := NewIndexHashedNodesCoordinator(arguments)
+	require.Nil(t, err)
+	epoch := uint32(1)
+
+	randomness := []byte("rand seed")
+
+	ihgs.nodesConfig[epoch] = ihgs.nodesConfig[0]
+
+	body := createBlockBodyFromNodesCoordinator(ihgs, epoch)
+	validatorsInfo, err := createValidatorInfoFromBody(body, arguments.Marshalizer, 10)
+	require.Nil(t, err)
+
+	err = ihgs.SetNodesConfigFromValidatorsInfo(epoch, randomness, validatorsInfo)
+
+	validators, err := ihgs.GetAllEligibleValidatorsPublicKeys(epoch)
+	require.Nil(t, err)
+	require.NotNil(t, validators)
+}
+
+func TestIndexHashedNodesCoordinator_SetNodesConfigFromValidatorsInfoMultipleEpochs(t *testing.T) {
+	t.Parallel()
+
+	arguments := createArguments()
+
+	ihgs, err := NewIndexHashedNodesCoordinator(arguments)
+	require.Nil(t, err)
+	epoch := uint32(1)
+
+	randomness := []byte("rand seed")
+
+	ihgs.nodesConfig[epoch] = ihgs.nodesConfig[0]
+
+	body := createBlockBodyFromNodesCoordinator(ihgs, epoch)
+	validatorsInfo, err := createValidatorInfoFromBody(body, arguments.Marshalizer, 10)
+	require.Nil(t, err)
+
+	ihgs.SetNodesConfigFromValidatorsInfo(epoch, randomness, validatorsInfo)
+	ihgs.SetNodesConfigFromValidatorsInfo(epoch+1, randomness, validatorsInfo)
+
+	_, ok := ihgs.nodesConfig[epoch+1]
```

Comment:
Why check internal member instead of using `IsEpochInConfig`?

Reply:
since this test is for `SetNodesConfigFromValidatorsInfo` i did not want to use another function if there is no direct dependency between them; if the tests will be refactored and moved in a separate "_test" package that would be very relevant, but in the current setup i would not interfere them (if IsEpochInConfig would change i would not want to affect this test also)